options/options.go

```diff
@@ -0,0 +1,23 @@
+package options
+
+type validationFunc func(val interface{}) bool
+
+type optionDetails struct {
+	defaultValue    interface{}
+	validationFuncs []validationFunc
+	help            string
+}
+
```

Comment:
Unless Translator exports defaults in a consumable way, avoiding this is not possible. We can change this later if Translator itself provides defaults.